vhost-device-scmi/CHANGELOG.md

```diff
@@ -9,6 +9,15 @@
 
 ### Deprecated
 
+## [0.2.0]
+
+## Changed
```

Comment:
Here it should be `### Changed`, but I already released it, we can fix next time ;-)
Just noticed while creating the release on github.

Reply:
Oops...

Reply:
fixed here: https://github.com/rust-vmm/vhost-device/pull/695 ;-)